packages/paste-core/components/sidebar/src/navigation/SidebarNavigation.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {styled, css} from '@twilio-paste/styling-library';
+import type {ThemeShape} from '@twilio-paste/theme';
+
+const SidebarNavigationWrapper = styled.div(({theme}: {theme: ThemeShape}) => {
+  const {colorBackgroundInverse, colorBackgroundInverseStronger} = theme.backgroundColors;
+
+  return css({
```

Comment:
suggestion: maybe worth a little comment here explaining this is the scrollbars and the overflow shadow effects.

Reply:
would be nice to have a story for showing the scrolling too - I don't think any are long enough